src/test/java/dk/cngroup/lentils/repository/UserRepositoryTest.java

```diff
@@ -41,11 +41,16 @@ public void saveTeamCreateNewUser() {
 
     @Test
     public void adminUserIsImportedOnStartup() {
-        Assert.assertEquals(1, userRepository.count());
         List<User> users = userRepository.findAll();
         Assert.assertEquals("admin", users.get(0).getUsername());
     }
 
+    @Test
+    public void organizerUserIsImportedOnStartup() {
+        List<User> users = userRepository.findAll();
+        Assert.assertEquals("organizer", users.get(1).getUsername());
```

Comment:
OPTIONAL: To same s tim poradim. Sla by udelat metoda `isUserInDb(String name)`, kterou by pak pouzivaly oba testy (`organizerUserIsImportedOnStartup` a `adminUserIsImportedOnStartup`).

Reply:
upraveno